R/makeMaps.R

```diff
@@ -0,0 +1,110 @@
+# makeMaps.R
+
+#' Generate the genetic interpretation map of BioGrid GGI tags of system's physical interactions.
+#'
+#' @return The dataframe mapping BioGrid GGI tag to its interpretation assuming that the system's components also interact physically.
```

Comment:
Needs to match the format
https://github.com/hyginn/BCB420.2019.ESA/blob/master/PullRequestReviewNotes.md

Reply:
Fixed.